src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -185,6 +185,7 @@
     <Compile Include="System\HandleTests.netstandard1.7.cs" />
     <Compile Include="System\AppContext\AppContext.Switch.cs" />
     <Compile Include="System\AppContext\AppContext.Switch.Validation.cs" />
+    <Compile Include="System\Reflection\StrongNameKeypairTests.netstandard1.7.cs" />
```

Comment:
Could you please fix the casing: StrongNameKeypairTests -> StrongNameKeyPairTests